architecture/additional_concepts/dynamic_admission_controllers.adoc

```diff
@@ -0,0 +1,313 @@
+[[architecture-additional-concepts-dynamic-admission-controllers]]
+= Dynamic Admission Controllers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+== Overview
+
+By default {product-title} uses standard, plugin-style 
```

Comment:
they are compiled in. That's the drawback. Customers cannot add their own without forking OpenShift. The new webhooks all to deploy admission controllers as webhooks in the cluster.